src/averageAges.js

```diff
@@ -36,8 +42,23 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people.filter(person => person.sex === 'f');
+
+  if (withChildren) {
```

Comment:
it's forbidden to use `if` in this task
but the solution with `Set` is nice :+1: 
try to get rid of `if`